msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -7,6 +7,153 @@ ms.author: anclear
 ---
 # Send proactive messages
 
+In some situations, a bot may need to send to the user a message that is not related to the current conversation. These types of messages are called **proactive messages**.
+
+Proactive messages can be useful in a variety of scenarios. These are some examples:  
+
+- The user previously asked the bot to monitor the price of a product, the bot can alert the user if the price of the product has changed.
+- A bot requires some time to compile a response to the user's question, it may inform the user of the delay and allow the conversation to continue in the meantime.
+
+## Best practices
+
+### Basic guidelines
+
+- Do not send several proactive messages within a short amount of time. Some channels enforce restrictions on how frequently a bot can send messages to the user, and will disable the bot if it violates those restrictions.
+- Ad hoc proactive message is the simplest type of proactive message. The bot simply interjects the message into the conversation whenever it is triggered, regardless if the user is engaged in a separate topic of conversation, The bot will not attempt to change the conversation in any way.
+- To handle notifications more smoothly, consider other ways to integrate the notification into the conversation flow, such as setting a flag in the conversation state or adding the notification to a queue.
+
+Proactive messages generally fall into one of the categories described below.
+
+### Welcome messages
```

Comment:
Our general welcome scenario is a mess, but for **Teams**, we can fix this. Again, ask Andrew what you need to add here to outline a best practice. (Can the bot send out a 1-1 message when a user adds the bot as an app? Can the bot send a group message when someone adds it to a team channel?)